apps/backoffice/src/components/forms/controllers/text-field-array-controller.tsx

```diff
@@ -119,6 +122,13 @@ export function TextFieldArrayController({
                     {...register(getFieldName(index))}
                     {...props}
                     margin="normal"
+                    sx={
```

Comment:
what about adding a `disabled` attribute?

Reply:
we want to keep the standard input style. add disable and change its style would be more complicated